packages/shared/components/Button.svelte

```diff
@@ -32,6 +32,7 @@
     export let events: Event<unknown>[] = []
 
     export let onClick: () => unknown = () => {}
+    export let isDebounceEnabled: boolean = true
```

Comment:
Don't add code that is not used anywhere :D. We can add something like this if there is a need for a button without a debounce